test/schema.py

```diff
@@ -235,7 +208,52 @@ def test_schema_changes(self):
     # check number of tables
     self._check_tables(shard_0_master, 5)
     self._check_tables(shard_1_master, 5)
-    self._check_tables(shard_2_master, 5)
+
+  def _setUp_tablets_shard_2(self):
```

Comment:
This is fine the way it is, but it could also be restructued so that this is a classSetup, and there are two test classes in this file (each with one test).